cmrtlib/linux/CMakeLists.txt

```diff
@@ -79,7 +79,8 @@ set(CMRT_DEFINES "")
 cmrt_include_directory(${CMAKE_CURRENT_LIST_DIR}/../agnostic)
 cmrt_include_directory(${CMAKE_CURRENT_LIST_DIR}/../linux)
 
-setup_library(igfxcmrt "${CMRT_SOURCES}" TRUE "igfxcmrt${TARGET_MODIFIER}")
+#setup_library(igfxcmrt "${CMRT_SOURCES}" TRUE "igfxcmrt${TARGET_MODIFIER}")
```

Comment:
This commit is already in a separate PR #413. Could you, please, drop the commit from the current PR?

Reply:
done, thanks